From c44c217e013062d4edf3099551f6c16fe5d595a8 Mon Sep 17 00:00:00 2001
From: make shi <b15407@freescale.com>
Date: Thu, 20 Sep 2012 17:10:20 +0800
Subject: [PATCH 2062/2463] ENGR00225131-03 MX6 USB: add platform_phy_power_on
 platform data in head file

IC designer had clarified that 1P1 can be turned off if we do not need support
remote wakeup. So If there is no requirement for USB remote wake up, the 1P1
can be turn off. USB driver will support dynamically turn on(off) 1P1 during
system suspend. 1P1 will be turn on depend on USB wakeup is enabled.

add platform_1p1_on platform data in head file.

Signed-off-by: make shi <b15407@freescale.com>
---
 include/linux/fsl_devices.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/linux/fsl_devices.h b/include/linux/fsl_devices.h
index 2a95573..ab22c4a 100644
--- a/include/linux/fsl_devices.h
+++ b/include/linux/fsl_devices.h
@@ -135,6 +135,7 @@ struct fsl_usb2_platform_data {
 	void (*platform_rh_suspend)(struct fsl_usb2_platform_data *);
 	void (*platform_rh_resume)(struct fsl_usb2_platform_data *);
 	void (*platform_set_disconnect_det)(struct fsl_usb2_platform_data *, bool);
+	void (*platform_phy_power_on)(void);
 
 	struct fsl_usb2_wakeup_platform_data *wakeup_pdata;
 	struct platform_device *pdev;
-- 
1.7.7.4

